apps/prairielearn/src/ee/pages/institutionAdminLti13/institutionAdminLti13.html.ts

```diff
@@ -339,6 +339,13 @@ ${JSON.stringify(instance.custom_fields, null, 3)}</textarea
       </form>
 
       <hr />
+      <p>
+        For testing, have the LMS admin configure their OpenID Connect Initiation URL to
+        <a href="/pl/lti13_instance/${instance.id}/auth/login?test">
+          /pl/lti13_instance/${instance.id}/auth/login?test
```

Comment:
To avoid any doubt, this is the line that I feel should have an `https://domain.tld` prefix included so that this can be a true copy/paste value for someone configuring an LTI instance.

Reply:
In my experience, Chrome makes relative links absolute when you copy to the clipboard. If you really think we should put the hostname in it then we can, but the functionality you're looking for is already the default.

Reply:
For configuration values, I want what the user _sees_ to match the correct value, not just what ends up in their clipboard.

Reply:
Fixed in a few places on that page in 421621e